test/nightly/global.resource.test.ts

```diff
@@ -31,7 +33,12 @@ suiteSetup(async function (this: Mocha.Context): Promise<void> {
         client = createAzureClient(testAccount.getSubscriptionContext(), CosmosDBManagementClient);
 
         // Create account
-        await Promise.all([delayCreateAccount(5, /graph/), delayCreateAccount(10, /MongoDB/), delayCreateAccount(15, /SQL/)]);
+        await Promise.all([
```

Comment:
We're already having problems with these nightly tests taking way too long. Maybe just keep the same number of accounts and only make one (Mongo or SQL) serverless instead of provisioned.

That being said, don't worry too much about the nightly tests. They're annoying to test locally and I think you're blocked from running them in our Azure Dev Ops instance. I'd be fine if you left this up to us.